JSON/genre.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Genres": [
+      {
+        "name": "pop"
+      }
+    ]
+  }
```

Comment:
As requested by the previous code reviewer when creating a game there is no way to add the first and last name of the author which means that data can't be saved in the author.json file if we can't create them, kindly fix that 🙏🏾
 Also after creating a game the data isn’t preserved in the game.json file. The multiplayer, last_played_at, publish_date, id date isn’t preserved kindly fix that and make sure all data can be preserved 🙃